daml-assistant/daml-sdk/src/main/scala/com/digitalasset/daml/sdk/SdkMain.scala

```diff
@@ -25,6 +26,7 @@ object SdkMain {
       case "extractor" => Extractor.main(rest)
       case "json-api" => JsonApi.main(rest)
       case "trigger-service" => TriggerService.main(rest)
+      case "oauth2-middleware" => Oauth2Middleware.main(rest)
```

Comment:
we should also publish standalone JARs for the trigger service and the middleware on GH releases, no need to include it in this PR.